lib/anycablebility/client.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+require 'thread'
+require 'json'
+require 'websocket-eventmachine-client'
+
+require 'anycablebility/waiter'
+
+module Anycablebility
+  # This is a simple ActionCable client created for testing purposes
+  class Client
+    attr_reader :state
+
+    def initialize(cable_url, debug = false, ignore_message_types = [])
```

Comment:
Let's move `debug` option into a general Anycablebility config.

We can also extract other options into a config and use it everywhere (e.g. `Anycablebility.config.redis_url`, etc).